index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Richard Portfolio Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <!-- header section -->
+    <header class="top_header">
+
+        <!-- nav menu bars -->
+        <nav class="top_nav">
+            <a href="#"><h3>Richard</h3></a>
+            <a href="#"><img src="./img/hamburger.png" alt="menu bar"></a>
+        </nav>
```

Comment:
- [ ] The logo is not supposed to be contained in the `nav` tag like you have done, you would find out in subsequent milestones and from the figma design the further contents of the nav but for now kindly put the logo as a stanalone positioned to the left and the nav positioned to the right using flexbox :+1:

Reply:
Thanks, this has been resolved @Efiamotu-1